tests/helpers/test_supplier_data_helpers.py

```diff
@@ -45,18 +46,37 @@ def _supplier_framework_response(self):
             return json.load(response_file)['supplierFrameworks']
 
     def test_suppliers_application_failed_to_framework(self, mocked_api_client):
-        supplier_framework = SupplierFrameworkDeclarations(mocked_api_client, mock.MagicMock(), dry_run=False)
+        supplier_framework = SupplierFrameworkDeclarations(
+            mocked_api_client,
+            mock.MagicMock(),
+            dry_run=False,
+            user='Data Retention Script {}'.format(datetime.now().isoformat())
+        )
         assert supplier_framework.suppliers_application_failed_to_framework('g-cloud-8') == [12345, 23456]
 
     def test_remove_declaration_from_suppliers(self, mocked_api_client):
-        mocked_api_client.remove_supplier_declaration.return_value = {'declaration': {}}
-        sfd = SupplierFrameworkDeclarations(mocked_api_client, mock.MagicMock(), dry_run=False)
-        assert sfd.remove_declaration(1, 'g-cloud-8')['declaration'] == {}
-        mocked_api_client.remove_supplier_declaration.assert_called_with(1, 'g-cloud-8', 'user')
+        with freeze_time('2019-01-01 12:00:00'):
+            mocked_api_client.remove_supplier_declaration.return_value = {'declaration': {}}
+            sfd = SupplierFrameworkDeclarations(
+                mocked_api_client,
+                mock.MagicMock(),
+                dry_run=False,
+                user='Data Retention Script {}'.format(datetime.now().isoformat())
+            )
+            assert sfd.remove_declaration(1, 'g-cloud-8')['declaration'] == {}
+            mocked_api_client.remove_supplier_declaration.assert_called_with(
+                1,
+                'g-cloud-8',
+                'Data Retention Script 2019-01-01T12:00:00')
```

Comment:
I assumed it would be passed through as the user, but if that's not the case then I'll need to add it to `SupplierFrameworkDeclarations.remove_declaration`

Reply:
Well, it depends if we need it or not. If not, we can simplify these tests a bit.

Reply:
I think we should - @benvand pointed out that it's the format we use elsewhere: https://github.com/alphagov/digitalmarketplace-scripts/blob/master/scripts/data-retention-remove-contact-information-data.py#L74